drivers/unix/dir_access_unix.cpp

```diff
@@ -405,14 +418,21 @@ bool DirAccessUnix::is_link(String p_file) {
 		return FAILED;
 
 	return S_ISLNK(flags.st_mode);
+#endif // VITA_ENABLED
 }
 
 String DirAccessUnix::read_link(String p_file) {
+#ifdef VITA_ENABLED
```

Comment:
Maybe reorganise this to have if defined(vita) || if defined(horizon)

Reply:
Would that have the same results? Then OK.

- [ ] Add `|| HORIZION_ENABLED` to this block